adagrams/game.py

```diff
@@ -1,11 +1,75 @@
+import random
 def draw_letters():
-    pass
+    '''returns a list of ten random letters as strings from a letter frequency pool'''
+    letters = ['A','A','A','A','A','A','A','A','A','B','B','C','C','D','D','D','D','E','E','E','E','E','E','E','E','E','E','E','E','F','F','G','G','G','H','H','I','I','I','I','I','I','I','I','I','J','K','L','L','L','L','M','M','N','N','N','N','N','N','O','O','O','O','O','O','O','O','P','P','Q','R','R','R','R','R','R','S','S','S','S','T','T','T','T','T','T','U','U','U','U','V','V','W','W','X','Y','Y','Z']
+    return random.sample(letters, 10)
 
 def uses_available_letters(word, letter_bank):
-    pass
+    '''takes in word and letter bank and returns true if every letter in word is available in letter_bank, returns false if not'''
+    letter_copy = letter_bank[:]
+    for letter in word:
+        if letter.islower():
+            letter = letter.upper()
+        if letter in letter_copy:
+            letter_copy.remove(letter)
+        else:
+            return False
```

Comment:
This works great! One small change: you can get rid of `if letter.islower()`. Your uppercasing logic works regardless of whether the letter is upper or lower case, so you don't need to explicitly check for it.